website/docs/plugins/ray_launcher.md

```diff
@@ -82,7 +85,7 @@ $ tree -L 1
 ├── model
 └── train.py
 
-$ python train.py hydra/launcher=ray_aws ray_mode=aws random_seed=1,2,3 -m
+$ python train.py hydra/launcher=ray_aws +ray_mode=aws random_seed=1,2,3 -m
```

Comment:
Can we get rid of ray_mode completely?

Since it's always used with ray_aws laucnher, what is the extra value it's providing? can't the things there just be inside the ray_aws launcher directly?

Reply:
Yea. I added this feedback in #666  :)

Reply:
ray_mode has some specific application config, like the `sync_down.include: ["*.pt", "*/"]` and `sync_up.include=["model", "*.py"]`.